tox.ini

```diff
@@ -65,17 +65,19 @@ commands =
     pylint enterprise
     pylint consent
     pylint integrated_channels
+    pylint enterprise_reporting
     pylint tests
     pylint test_utils
     pylint --py3k enterprise
     pylint --py3k consent
     pylint --py3k integrated_channels
+    pylint --py3k enterprise_reporting
     pylint --py3k tests
     pylint --py3k test_utils
     rm tests/__init__.py
-    pycodestyle enterprise consent integrated_channels tests test_utils
-    pydocstyle enterprise consent integrated_channels tests test_utils
-    isort --check-only --recursive tests test_utils enterprise consent manage.py setup.py test_settings.py
+    pycodestyle enterprise consent integrated_channels enterprise_reporting tests test_utils
+    pydocstyle enterprise consent integrated_channels enterprise_reporting tests test_utils
+    isort --check-only --recursive tests test_utils enterprise consent integrated_channels enterprise_reporting manage.py setup.py test_settings.py
```

Comment:
Ah wow, didn't know we weren't checking for this. Can you also add these to the `isort` make target?